### PR TITLE
search 7.1 | CCR for 7.1 | LRDOCS-7722

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-7722
added a commit as per https://github.com/lipusz/liferay-docs/pull/71

### CODE REVIEW

**Comment by lipusz** [comment]
Hello Russ,

I was wondering if you left out the bullet list of the les features/modules intentionally.

I'm referring to this section:
> Detailed installation and usage instructions are available in the article for each LES feature, including

Cross Cluster Replication

Learning to Rank

Monitoring Elasticsearch

Securing Elasticsearch [Free without LES for Liferay CE and DXP with Elasticsearch 7]

**Comment by rbohl** [comment]
Ahh sorry, I can add that in--I thought you meant the text content not necessarily the TOC

**Comment by lipusz** [comment]
:+1: LGTM with that note.

File: en/discover/deployment/articles-dxp/02-installing-search-engine/02-enterprise-search/00-intro.markdown [hunk 1]
```diff
@@ -6,39 +6,46 @@ header-id: installing-x-pack
...
+license, but you can enable a
+[30-day trial](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/start-trial.html)
+to work with in the meantime.
+
+Detailed installation and usage instructions are available in the article for
+each LES feature, including
+
+-  [Liferay Learn] [Cross-Cluster Replication](https://learn.liferay.com/dxp/latest/en/using-search/liferay-enterprise-search/cross-cluster-replication/cross-cluster-replication.html)
+-  [Monitoring Elasticsearch](/docs/7-1/deploy/-/knowledge_base/d/monitoring-product)
+-  [Securing Elasticsearch](/docs/7-1/deploy/-/knowledge_base/d/installing-liferay-enterprise-search-security) [Free without LES for Liferay CE and DXP with Elasticsearch 7]
```
[the remark this answers is not in the document]
Comment (rbohl) [reply en/discover/deployment/articles-dxp/02-installing-search-engine/02-enterprise-search/00-intro.markdown:33]: Interesting--I guess I should add this piece of information (ES 7 on Liferay 7.1 is DXP-only) to the "Installing Elasticsearch 7" and "Upgrading to Elasticsearch 7" articles as well. I'll add a commit and send this up to Rich.

**Pull request merged**